operatorcourier/validate.py

```diff
@@ -237,3 +248,152 @@ def _type_validation(self, bundleData, typeName, validator, required):
             valid = False
 
         return valid
+
+    def _ui_validation_ocp(self, bundleData):
+        valid = True
+        logger.info("Validating cluster service versions for OCP Operator Hub UI.")
```

Comment:
This looks like a placeholder, will this be implemented in a future PR?

Reply:
That's the plan